sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java

```diff
@@ -155,9 +197,14 @@ public QualitygatesService qualityGates() {
     return qualityGatesService;
   }
 
+  @Override
+  public org.sonarqube.ws.client.measure.MeasuresService measuresOld() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "MeasuresService"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)